tests/bdd/features/cordon/node/feature.feature

```diff
@@ -0,0 +1,48 @@
+Feature: Cordoning
+
+  Background:
+    Given Multiple uncordoned nodes
+
+  Scenario: Cordoning a node
+    Given an uncordoned node
+    When the node is cordoned
+    Then new resources of any type cannot be scheduled on the cordoned node
+
+  Scenario: Cordoning a cordoned node
+    Given a cordoned node
+    When the cordoned node is cordoned
+    Then the cordoned node should remain cordoned
+    And new resources of any type cannot be scheduled on the cordoned node
+
+  Scenario: Cordoning a node with existing resources
+    Given an uncordoned node with resources
+    When the node is cordoned
+    Then new resources of any type cannot be scheduled on the cordoned node
+    And existing resources remain unaffected
+
+  Scenario: Uncordoning a node
+    Given a cordoned node
+    When the node is uncordoned
+    Then new resources of any type can be scheduled on the uncordoned node
+
+  Scenario: Uncordoning an uncordoned node
+    Given an uncordoned node
+    When the uncordoned node is uncordoned
+    Then the uncordoned node should remain uncordoned
+    And new resources of any type can be scheduled on the uncordoned node
+
+  Scenario: Uncordoning a node with existing resources
+    Given a cordoned node with resources
+    When the node is uncordoned
+    Then new resources of any type can be scheduled on the uncordoned node
+    And existing resources remain unaffected
```

Comment:
Do we need this use case, what does unaffected mean here?

Reply:
I was trying to highlight that resources that are on the cordoned node are unaffected by an uncordon operation. Do you think this is unnecessary?

Reply:
Not sure, just seems a bit vague - eg: I wouldn't know how to write the fixture for this clause.

Reply:
Fair enough - I've remove it.